trk/src/Navigation/AppNavigator.js

```diff
@@ -35,6 +36,7 @@ function ProfileStack() {
       <Stack.Screen name="Profile" component={UserProfile} />
       <Stack.Screen name="Detail" component={ClimbDetailScreen} />
       <Stack.Screen name="Set" component={SetDetail} />
+      <Stack.Screen name="Your Settings" component={Settings} />
```

Comment:
Naming conventions would have the Stack Screen be called: 'Settings', instead of 'Your Settings'.

### Strava
![0E3C694E-2A1D-4F15-AC95-D451904F5744_1_102_o](https://github.com/nagimonyc/trk-app/assets/66965648/b1a30005-35c6-4d50-9d9b-91be727dcab7)

### ChatGPT
![E5EAB1B3-F15D-4EE1-8019-F1EFF4381ECB_1_102_o](https://github.com/nagimonyc/trk-app/assets/66965648/fa1a1493-2afb-4195-9d21-805e145b6682)

### NFC Tools
![8BFA37A6-2062-47C8-988D-4D83343AB2CB_4_5005_c](https://github.com/nagimonyc/trk-app/assets/66965648/2ed22baa-6ddc-45a5-9ac1-1eac527cfcd5)


⚠️ Changing this name field will have repercussions on your 'navigation.navigate'.